cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -359,6 +397,45 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
         statsOnlyResultTable()
       }
     }
+    section("Listing database aliases", "administration-databases-show-alias") {
```

Comment:
We need to add this to somewhere (I think in the manual-modelling repo) to get it to show up in the left menu